firstvoices/backend/serializers/category_serializers.py

```diff
@@ -1,21 +1,54 @@
 from rest_framework import serializers
 
 from backend.models.category import Category
+from backend.models.constants import CATEGORY_POS_MAX_TITLE_LENGTH
+from backend.serializers.base_serializers import (
+    SiteContentLinkedTitleSerializer,
+    UpdateSerializerMixin,
+)
+from backend.serializers.validators import HasNoParent, SameSite, UniqueForSite
 
```

Comment:
This diff is a mess because I renamed some classes. Sorry! 

1. Added the LinkedCategorySerializer, for listing related categories on the detail view
2. Renamed the 2 list serializers (child and parent) and added url fields.
3. Added validators 
4. Changed the parent field to have a read_only version (with a linked title object) and a write_only version (just the uid)